homeworks/serhii.ripchanskyi_Ripchanskiy/11-dependency-injection/vechicle/src/app/app.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { finalize } from 'rxjs/operators';
+
+import { Vehicle } from './models/';
+import { VehicleService } from './services';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  title = 'vehicle';
+  vehicles: Vehicle[] = [];
+  areVehiclesLoading: boolean = false;
+
+  constructor(private vehicleService: VehicleService) {
+    this.getVehicles();
```

Comment:
Sorry, forgot to say, that you need to call such requests in the `ngOnInit` hook.
So please, add implement the `OnInit` hook and use this `this.getVehicles();` call inside it